scripts/docker/Dockerfile

```diff
@@ -69,6 +69,8 @@ RUN \
 
 RUN apt-get autoremove
 
+ENV LC_ALL C.UTF-8
```

Comment:
Helped a little bit, but couldn't manage to build with emscripten using our linux docker.
More investigation needed.